src/components/TopBar/TopBar.tsx

```diff
@@ -61,7 +60,8 @@ export const TopBar: React.FunctionComponent<TopBarProps> & {
   contextControl,
 }: TopBarProps) {
   const i18n = useI18n();
-  const {logo} = useTheme();
+  // TODO: This behavior will be re-enabled in a separate PR.
+  const logo = undefined as {[key: string]: any} | undefined;
```

Comment:
Drawing attention here to ensure this is capture in the following PR: https://github.com/Shopify/polaris-react/pull/4667